### PR TITLE
make sentinel class for  nbformat NO_CONVERT

### PR DESCRIPTION
**Pull request by Carreau**
Cf discussion in #7876 
Make Signature of nbconvert writes and a few others a little more sane.

### CODE REVIEW

File: jupyter_nbformat/__init__.py [hunk 1]
```diff
@@ -40,7 +40,17 @@ class NBFormatError(ValueError):
     pass
 
 # no-conversion singleton
-NO_CONVERT = object()
+class Sentinel(object):
+    """Prevent nbformat to convert notebooks to most recent version.
```
Comment (takluyver) [inline jupyter_nbformat/__init__.py]: The docstring is really for the instance, not the class. Maybe it should have a second argument to take a description?

Comment (Carreau) [inline jupyter_nbformat/__init__.py]: Done (push in progress)

File: IPython/html/static/components [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 031e6c979ffd52c457a6ca8427116d9b2120e5b5
+Subproject commit e7f5a5a85f5dc0173e0b6ebacaa38c2af75a8195
```
Comment (minrk) [inline IPython/html/static/components]: Did you mean to update components?

**Comment by Carreau** [comment]
updated

```
In [2]: BaseFormatter.pop?
Signature: BaseFormatter.pop(self, typ, default=IPython.core.formatters._raise_key_error)
```

```
In [3]: from jupyter_nbformat import writes
In [4]:writes?
Signature: writes(nb, version=jupyter_nbformat.NO_CONVERT, **kwargs)
```

```
In [5]: from traitlets.traitlets import TraitType

In [6]: TraitType?
Init signature: TraitType(self, default_value=traitlets.traitlets.NoDefaultSpecified, allow_none=None, **metadata)
```

**Comment by takluyver** [comment]
+1

**Comment by Carreau** [comment]
Thanks !

**Pull request merged**